util/system/path.go

```diff
@@ -5,10 +5,10 @@ package system
 // ':' character .
 const DefaultPathEnvUnix = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
 
-// DefaultPathEnvWindows is windows style list of directories to search for
-// executables. Each directory is separated from the next by a colon
-// ';' character .
-const DefaultPathEnvWindows = "c:\\Windows\\System32;c:\\Windows"
+// DefaultPathEnvWindows is deliberately empty on Windows and the default path
+// must be set by the image. BuildKit has no context of what the default
+// path should be.
+const DefaultPathEnvWindows = ""
 
 func DefaultPathEnv(os string) string {
```

Comment:
Seeing as how the empty string is (almost?) always a special case, I think https://github.com/golang/go/wiki/CodeReviewComments#in-band-errors applies here.
```suggestion
func DefaultPathEnv(os string) (string, bool) {
```

Reply:
updated; added the `bool` return.